mobile-academy/src/main/java/org/motechproject/nms/mobileacademy/domain/course/LessonContentLesson.java

```diff
@@ -0,0 +1,43 @@
+package org.motechproject.nms.mobileacademy.domain.course;
+
+import org.motechproject.mds.annotations.Entity;
+import org.motechproject.mds.annotations.Field;
+
+/**
+ * Contains details about actual content files to be played while playing a lesson.
+ */
+@Entity
+public class LessonContentLesson {
+
```

Comment:
Having classes called Lesson, LessonContent, and LessonContentLesson is confusing for me. :) Could LessonContentLesson be LessonContentDetails or something similar?